src/main/java/org/opentripplanner/analyst/request/SampleFactory.java

```diff
@@ -72,12 +155,24 @@ public Sample findClosest(List<Edge> edges, Coordinate pt, double xscale) {
         Candidate c = new Candidate();
         // track the best geometry
         Candidate best = new Candidate();
+
         for (Edge edge : edges) {
             /* LineString.getCoordinates() uses PackedCoordinateSequence.toCoordinateArray() which
              * necessarily builds new Coordinate objects.CoordinateSequence.getOrdinate() reads them 
              * directly. */
             c.edge = edge;
             LineString ls = (LineString)(edge.getGeometry());
+
+            // We used to require samples to link to OSM vertices, but that means that
+            // walking to a transit stop adjacent to the sample requires walking to the
+            // end of the street and back.
+
+            // However, linking to splitter vertices means that this sample can only be egressed in one direction,
+            // because the splitter vertex is on one half of a bidirectional edge. Additionally, the splitter vertices and the connected edges
```

Comment:
We should probably say "bidirectional edge pair" lest someone in the future think there are some magic bidirectional edges.